docs/api/utilities/suggestion.md

```diff
@@ -21,6 +21,11 @@ Allows or disallows spaces in suggested items.
 
 Default: `false`
 
+### allowedPrefixes
```

Comment:
Since this talks about `prefixes` I expected an array of strings - not only one string. How is this supposed to be used when I want to allow multiple prefixes? Do I enter them in one string with spaces separated?

Reply:
Since you use it in the regex - maybe we should make this clear in the docs?

Reply:
Ah I see - I just wasn't sure how this would've been used as a string-only which is also why I thought about adding a note how this string is used.

I like the approach of an array of strings more to make it clear that each string in this regex is matched separated